### DISCUSSION TITLE
请问list2lt最后一行最后一个单元格假如无内容，该怎么写才能正常显示

### DISCUSSION THREAD

**Discussion by hsingll** [any-block/any-block#53] [Q&A] [answered]
目前不能解析最后一个`| `
<img width="644" alt="Snipaste_2024-09-02_17-07-55" src="https://github.com/user-attachments/assets/73e98df6-f2c3-47a8-a4ee-b08c0649a48a">

**Comment by LincZero** [comment]
1. 可以不写最后一个 `|`，缺点是这样的话右边是空缺的，不建议强迫症使用
2. 拆分正则：`inline_split: /\| |,  |， |\.  |:  |： /`，所以最后一个符号可以是：
  - `|`+空格
  - 中文全角符号+空格
  - 英文半角符号+两空格
3. 为什么要 `|`+`空格`，而不能只是 `|`？为了避免误触发，以及保证里面可以嵌套表格，避免这种情况发生冲突

**Reply by hsingll** [reply]
我目前是`| `，就是`|`加空格。如图里那样。

```
[list2lt]
- < 字段结构| 属性| 描述| 备注
- `LVC_S_LAYO`| | | 
	- 引用`ALV_S_LAYO`| | | 
	- 引用`LVC_S_L001`| | | 
		- `STYLEFNAME`| `TYPE LVC_FNAME` `CHAR(30)`| 样式码来源字段| 见[[字段格式]]
	- 引用`LVC_S_L002`| | | 
		- `NO_ROWMARK`| `TYPE LVC_ROWMK` `CHAR(1)`| 禁用行选择| 
```
<img width="644" alt="Snipaste_2024-09-02_17-07-55" src="https://github.com/user-attachments/assets/73e98df6-f2c3-47a8-a4ee-b08c0649a48a">

**Comment by LincZero** [comment]
可以啊

![图片](https://github.com/user-attachments/assets/95975139-9142-46a6-8c37-4b0e8802a58e)

你再检查下是不是真的有空格

**Reply by LincZero** [reply]
哦，我知道了，阅读模式把尾部的空格给去除掉了

**Reply by LincZero** [reply, marked as answer]
3.1.3-beta 已修复
